extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
Ah, ok, I can change them to be dev errors then. But I'd like to see is something like a first mark survives. So, if something has been marked as a user error in some inner code, trying `dev().mabyeMark()` can still provide context but doesn't change the user mark.